setup.py

```diff
@@ -42,7 +42,7 @@
     license="Apache 2",
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
-    install_requires=["pytket ~= 1.5", "iqm-client == 4.3"],
+    install_requires=["pytket ~= 1.5", "iqm-client == 9.1"],
```

Comment:
```suggestion
    install_requires=["pytket ~= 1.5", "iqm-client ~= 9.1"],
```
Can we do this, assuming 9.x will be API-compatible? (Question for IQM team.)

Reply:
Can we also update the pytket dependency to `~=1.7`?

Reply:
>Can we do this, assuming 9.x will be API-compatible? (Question for IQM team.)

Yes, 9.x will be API-compatible.